docs/ui/components/AskPageAI/AskPageAIChat.tsx

```diff
@@ -0,0 +1,440 @@
+        return;
+      }
+      if (feedbackTarget.reaction === reaction) {
+        return;
+      }
+      addFeedback(feedbackTarget.id, reaction);
+    },
+    [addFeedback, feedbackTarget]
+  );
+
+  const handleNavigation = useCallback(
+    (event?: MouseEvent<HTMLAnchorElement>) => {
+      if (event?.defaultPrevented) {
+        return;
+      }
+      onMinimize?.();
+    },
+    [onMinimize]
+  );
+
+  const markdownComponents = useMemo(() => {
+    return {
+      code({
+        inline,
+        children,
+      }: {
+        inline?: boolean;
+        children?: React.ReactNode;
+        className?: string;
+      }) {
+        const childArray = Children.toArray(children);
+        const raw = childArray.map(node => (typeof node === 'string' ? node : '')).join('');
+        const clean = raw.replace(/\n$/, '');
+
+        const tokenCount = clean.trim().split(/\s+/).length;
+        const hasLineBreak = clean.includes('\n');
+
+        if (inline) {
+          const trimmed = clean.replace(/^`+/, '').replace(/`+$/, '');
+          const display = trimmed.length > 0 ? trimmed : clean;
+          const shouldRenderInline = display.trim().split(/\s+/).length <= 1;
+
+          if (shouldRenderInline) {
+            return (
+              <code className="rounded bg-subtle px-1 py-0.5 text-xs text-default">{display}</code>
+            );
+          }
+
+          return <MarkdownCodeBlock code={display.trim()} compact />;
+        }
+
+        if (!hasLineBreak && tokenCount <= 1) {
+          return (
+            <code className="rounded bg-subtle px-1 py-0.5 text-xs text-default">
+              {clean.trim()}
+            </code>
+          );
+        }
+
+        return <MarkdownCodeBlock code={clean} />;
+      },
+      a({ children, href, ...rest }) {
+        return (
+          <a
+            {...rest}
+            href={href ?? '#'}
+            className={mergeClasses('text-link', rest.className)}
+            onClick={event => {
+              rest.onClick?.(event);
+              if (event.defaultPrevented || event.metaKey || event.ctrlKey || event.button !== 0) {
+                return;
+              }
+              handleNavigation(event);
+            }}>
+            {children}
+          </a>
+        );
+      },
+    } as Components;
+  }, [handleNavigation]);
+
+  return (
+    <div className="flex h-full flex-col overflow-hidden bg-default">
+      <div className="flex items-center justify-between border-b border-default bg-palette-black px-4 py-3 text-palette-white">
+        <div className="flex items-center">
+          <span
+            className={mergeClasses(
+              'inline-flex size-8 items-center justify-center rounded-full bg-palette-white shadow-xs'
+            )}
+            style={headerAccentBackground}>
+            <Stars03DuotoneIcon className="icon-sm text-palette-white" />
+          </span>
+          <span className="text-sm font-medium leading-tight text-palette-white">
```

Comment:
There should be a space between the icon and title

Reply:
Fixed:

<img width="618" height="220" alt="CleanShot 2025-09-26 at 18 58 24" src="https://github.com/user-attachments/assets/cc0acc67-1942-403a-8d34-88f43564cad9" />